### ISSUE TITLE
Add support for ASP.net MVC 5

### ISSUE THREAD

**Issue by ghost** [johnpapa/HotTowel#18]
I'd like to see HotTowel support for the new VS 2013 Web ASP.net MVC 5 project directly.
I tried to use the NuGet package and see if I could get it to work but the references aren't updated to EntityFramework 6 and I got blocked at that stage.

**Comment by jkolonko** [comment]
I think now all the dependent packages are now compatible with EF6 and MVC5 (including Breeze). I updated my project to MVC5 manually, only HotTowel dependency prevent me from upgrading to EF6.

**Comment by gitmar** [comment]
I think HotTowel, as framework of libraries's assemblies, must develop fluid dependencies. Like @jkolonko  I'm so blocked by this last one. Anyone know a workaround (not passing by angular)?

**Comment by jkolonko** [comment]
My workaround was simple. I just uninstalled HotTowel package (without dependencies) and upgraded all libraries manually.

**Comment by gitmar** [comment]
Yes, its works. The difficult was because I'd tried to mix Hottowel template with asp authentication. It would be perfect if the future Hottowel integrate authentication module.

**Comment by mwpowellhtx** [comment]
Any update on this issue? Updated to MVC 5 (minimum) ? Don't care about EF6, per se; I want to keep my data dependencies separate from the application views, per se.

**Comment by johnpapa** [comment]
PRs are welcome
